python/ml4ir/base/model/architectures/dnn.py

```diff
@@ -2,6 +2,14 @@
 from tensorflow.keras import layers
 from typing import List
 
+from ml4ir.base.features.feature_config import FeatureConfig
+from ml4ir.base.features.feature_fns.categorical import get_vocabulary_info
+from ml4ir.base.io.file_io import FileIO
+
+
+OOV = 1
```

Comment:
@Ullimague Is it okay to hard code this?

Reply:
Yes! It basically provides free "documentation", and we directly understand:
`layer_args["units"] = len(vocabulary_keys) + OOV`
whereas
`layer_args["units"] = len(vocabulary_keys) + 1`
we don't know what the + 1 is about. 

On top of that OOV could be reuse elsewhere if needed.